src/components/m-header-logo/js/header-logo.js

```diff
@@ -10,7 +10,8 @@ class HeaderLogo {
     link: '.js-header-logo__link',
   }
 
-  constructor(wcNode, options = {}) {
+  constructor(wcNode, options = {
```

Comment:
oh man `eslint --fix`really seems to have issues with default values of empty lits

Reply:
I don't have a strong opinion on this. I have a stronger opinion about useless warnings in outputs, like `npm run test` which was polluted with tons of warnings.

Please reconfigure your wished linter configs, and we can rerun the auto fix.